installer/conf/container.conf

```diff
@@ -52,7 +52,7 @@
 
 # Container log
 <source>
-	type custom_sudo_tail
+	type containerlog_sudo_tail
```

Comment:
Add a sample line from the file so your "format /\"log\"=>\"(?<LogEntry>.*)", \"stream\"=>\"(?<LogEntrySource>.*)", \"time\"=>\"(?<TimeGenerated>.*)", \"ContainerID\"=>\"(?<Id>.*)", \"Image\"=>\"(?<Image>.*)", \"Name\"=>\"(?<Name>.*)", \"SourceSystem\"=>\"(?<SourceSystem>.*)"}/" is easy to understand.